sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/implementation/helpers/BoundingBoxPropertiesHelper.java

```diff
@@ -0,0 +1,22 @@
+package com.azure.maps.render.implementation.helpers;
+
+import com.azure.maps.render.implementation.models.BoundingBoxPrivate;
+import com.azure.maps.render.models.BoundingBox;
+
+public final class BoundingBoxPropertiesHelper {
+    private static BoundingBoxAccessor accessor;
+
+    private BoundingBoxPropertiesHelper() { }
+
+    public interface BoundingBoxAccessor {
+        void setToBoundingBoxPrivate(BoundingBox boundingBox, BoundingBoxPrivate boundingBoxPrivate);
+    }
+
+    public static void setAccessor(final BoundingBoxAccessor boundingBoxAccessor) {
+        accessor = boundingBoxAccessor;
+    }
+
```

Comment:
I don't think this is being used right. 

Typically, the pattern is to set a Public instance of a class using the properties of a Private instance

BoundingBox b = new BoundingBox();
b.setFromBoundingBoxPrivate(boundingBoxPrivate);

It seems like you are doing the opposite. TO convert from public to private you don't need a properties helper at all. Just code in the Utility method.